napari/layers/points/_tests/test_points_key_bindings.py

```diff
@@ -1,8 +1,12 @@
 from napari.layers import Points
 
 
-def test_select_all():
+def test_select_all(make_napari_viewer):
     """Test select all key binding."""
+
+    # make viewer to bind shortcuts
+    viewer = make_napari_viewer()  # noqa: F841
+
```

Comment:
What do I need to do in order to use the `make_napari_viewer` fixture?  Locally, this test passes.  I don't understand how to make it work here...

Reply:
It's the headless test that's failing, which means you've put something that requires qt (make_napari_viewer) in a part of the code base that otherwise wasn't depending on at at all.

If this test can't be redesigned to be more of a unit test (ie, not use qt and the viewer), then just move this test to the top level tests folder, or one of the modules that uses qt

Reply:
The keybindings are bound in the qt viewer now, so that was the reason adding the viewer.  I was trying to avoid duplicating code that binds the shortcuts from settings, otherwise, I don't really need the viewer.  I am going to move this test, but if you think that the binding should happen somewhere else or I should do things differently, let me know.  Are the keybindings going to be used outside the viewer anyway?

Reply:
Maybe we can just test that the functions themselves run successfully? I.e. not test the actual keybinding here - so more like the tests in https://github.com/napari/napari/blob/master/napari/layers/shapes/_tests/test_shapes_key_bindings.py

Reply:
@sofroniewn , OK, I can take a look!

Reply:
@sofroniewn I added tests for all the points functions.  I also moved that other test to the qt_viewer tests.  Not sure its needed, and I can remove it if not.

Reply:
Love the tests for all the point functions!! So great. 

> I also moved that other test to the qt_viewer tests. Not sure its needed, and I can remove it if not.

I'd probably drop it, no more need i think

Reply:
OK, I took it out.   :)